src/core_cli/Core_CLI.ml

```diff
@@ -324,6 +324,10 @@ let dump_rule file =
   let rules = Parse_rule.parse file in
   rules |> List.iter (fun r -> pr (Rule.show r))
 
+let dump_contributions () =
```

Comment:
I recently moved the dumpers to Core_actions.ml, so you'll get a merge conflict, but just move your function there
and you should be fine.

Reply:
Moved to `Core_actions.ml`